goblet/handlers/handler.py

```diff
@@ -72,3 +72,8 @@ def _check_or_enable_service(self, enable=False):
         if not self.resources:
             return
         return check_or_enable_service(self.required_apis, enable)
+
+    def get_resource_type(self):
```

Comment:
not sure if this makes sense. i know we use it 
```
            resource_type = v.get_resource_type()
            if resource_type and resource_type != "http":
```

but maybe we can just rewrite to  `if v.resource_type != "http" and v.resources:`